src/tests/unit/tests/bug-filing/common/issue-details-builder.test.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { It, Mock, MockBehavior } from 'typemoq';
+import { createIssueDetailsBuilder } from '../../../../../bug-filing/common/create-issue-details-builder';
+import { MarkupFormatter } from '../../../../../bug-filing/common/markup/markup-formatter';
+import { EnvironmentInfo } from '../../../../../common/environment-info-provider';
+
+describe('Name of the group', () => {
+    const sampleIssueDetailsData = {
+        pageTitle: 'pageTitle<x>',
+        pageUrl: 'pageUrl',
+        ruleResult: {
+            failureSummary: 'RR-failureSummary',
+            guidanceLinks: [{ text: 'WCAG-1.4.1' }, { text: 'wcag-2.8.2' }],
+            help: 'RR-help',
+            html: 'RR-html',
+            ruleId: 'RR-rule-id',
+            helpUrl: 'RR-help-url',
+            selector: 'RR-selector<x>',
+            snippet: 'RR-snippet   space',
+        } as any,
+    };
+    const environmentInfo: EnvironmentInfo = {
+        extensionVersion: '1.1.1',
+        axeCoreVersion: '2.2.2',
+        browserSpec: 'test spec',
+    };
+
+    const markupMock = Mock.ofType<MarkupFormatter>(null, MockBehavior.Strict);
+
+    beforeEach(() => {
+        markupMock.reset();
```

Comment:
As per our engineering discussion (?), should we not be creating the Mock here instead of having to reset?

Reply:
our engineering discussion was about not using `after` to run verify on our mocks... I don't think we land on anything about the use of `reset` or where to create the mocks

Reply:
updated